README.md

```diff
@@ -1,4 +1,10 @@
-## UnifyCR: A Distributed Burst Buffer File System - 0.1.0
+# UnifyCR: A Distributed Burst Buffer File System - 0.1.0
+
+## Build Status
+The current status of the UnifyCR dev branch is
+[!Build Status](https://api.travis-ci.org/LLNL/UnifyCR.png?branch=dev)]
```

Comment:
I think you're missing a bracket after the exclamation point:
```
[![Build Status](https://api.travis-ci.org/LLNL/UnifyCR.png?branch=dev)]
```

Reply:
@nedbass thanks, added now.

Reply:
Hmm the markdown is still not rendering quite right. Maybe the URL part needs to be on the same line.

Reply:
@nedbass okay, it looks like it is rendering correctly now.